### PR TITLE
fix: [PL-22100]: UI changes for SAML settings client secret

### PR DESCRIPTION
**Pull request by pbarapatre10**
##### Summary:

Azure SAML - Handle client secret as password instead of secret-ref for NG

##### Jira Links:

[](https://harness.atlassian.net/browse/PL-22100)

##### Screenshots:

image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by pbarapatre10** [comment]
retrigger titlecheck

**Comment by pbarapatre10** [comment]
retrigger titlecheck

**Comment by pbarapatre10** [comment]
retrigger prettier

**Comment by pbarapatre10** [comment]
Added a task in JIRA to reduce cognitive complexity of the SAMLProviderForm https://github.com/harness/harness-core-ui/pull/6958

File: src/modules/33-auth-settings/modals/SAMLProvider/views/SAMLProviderForm.tsx [hunk 1]
```diff
@@ -294,7 +283,11 @@ const SAMLProviderForm: React.FC<Props> = ({ onSubmit, onCancel, samlProvider })
                               {Providers.AZURE === values?.samlProviderType && values.enableClientIdAndSecret && (
                                 <Container width={300} margin={{ top: 'medium' }}>
                                   <FormInput.Text name="clientId" label={getString('common.clientId')} />
-                                  <SecretInput name="clientSecret" label={getString('common.clientSecret')} />
+                                  <FormInput.Text
```
Comment (avinashmadhwani02) [inline src/modules/33-auth-settings/modals/SAMLProvider/views/SAMLProviderForm.tsx:286]: why are we not using SecretInput?

**Review by meenakshi-harness — APPROVED** [review]

File: src/modules/33-auth-settings/modals/SAMLProvider/views/SAMLProviderForm.tsx (hunk 1 above)
Comment (pbarapatre10) [reply src/modules/33-auth-settings/modals/SAMLProvider/views/SAMLProviderForm.tsx:286]: In 'ng' we can't use secret input as NG here just acts as a relay. SAML setting can only be one in an account and manageable through both CG/NG. The actual secret refs are created in back-end in CG side. Thanks

**Review by avinashmadhwani02 — APPROVED** [review]

**Pull request merged**